# Description: Remove shebang (#!) from .py files to comply with policy.
# Author: Kenneth J. Pronovici <pronovic@debian.org>
--- a/epydoc/gui.py
+++ b/epydoc/gui.py
@@ -1,5 +1,3 @@
-#!/usr/bin/env python
-#
 # objdoc: epydoc command-line interface
 # Edward Loper
 #
